lib/ansible/modules/systemd.py

```diff
@@ -439,18 +439,22 @@ def main():
             (rc, out, err) = module.run_command("%s is-enabled '%s'" % (systemctl, unit))
             masked = out.strip() == "masked"
 
+            if module.params['masked']:
```

Comment:
I'm also not really sure about that. I think if service is masked, it might be a reason for it, and we shouldn't just enable masked service until it's explicitly told, so user should be aware what exactly he is doing. And it's kind of current behaviour (except it's not fully working). As generally services doesn't appear masked - they are explicitly told to take that state.

Reply:
Ok. Makes sense.